inst/stan/functions/observation_model.stan

```diff
@@ -195,6 +195,29 @@ vector report_log_lik(array[] int cases, vector reports,
   return(log_lik);
 }
 
+
+/**
+ * Custom safe version of the negative binomial sampler
+ *
+ * This function generates random samples of the negative binomial distribution
+ * whilst avoiding numerical overflows.
+ *
+ * @param mu Real value ofr mean mu.
+ * @param phi Real value for phi.
+ *
+ * @return A random sample from the negative binomial distribution.
```

Comment:
Approximately neg binomial right because I see we sample from Poisson given the size of mu?

Reply:
have added documentation in 285aac8